contents/docs/migrate/migrate-from-amplitude.mdx

```diff
@@ -10,65 +10,218 @@ import MigrationWarning from "./_snippets/migration-warning.mdx"
 
 Migrating from Amplitude is a two step process:
 
-1. Export your data from Amplitude using the [Amplitude Export API](https://www.docs.developers.amplitude.com/analytics/apis/export-api/i).
+1. Export your data from Amplitude using the [Amplitude Export API](https://amplitude.com/docs/apis/analytics/export).
 
-2. [Import data into PostHog](/docs/migrate) using PostHog's Python SDK or `batch` API with the `historical_migration` option set to `true`.
+2. [Import data into PostHog](/docs/migrate) using PostHog's [Python SDK](/docs/libraries/python) or [`batch` API](/docs/api/capture) with the `historical_migration` option set to `true`.
```

Comment:
Do I need to use the Python SDK specifically? or can I use node ruby etc. If not, why not? Should we include the reason why not?

Reply:
These ones have support for historical exports and the others don't